example_train.py

```diff
@@ -59,12 +59,15 @@
 tokenizer = lmp.tokenizer.CharTokenizer()
 
 # 讓使用者決定是否 uncase
-uncaseOrNot = True if len(sys.argv) > 1 and sys.argv[1].lower() == 'uncase'   else False
+parser = argparse.ArgumentParser()
+parser.add_argument("-u", "--uncase", help="regard capital letter and lowercase letter as same word",
```

Comment:
Do not use short-hand argument (i.e. `"-u"`).
Change helper text into `"convert all upper case into lower case."`.
Indicate `--uncase` as required argument.

Reply:
rename `arg.uncase` into `arg.is_uncased` for consistency